### PR TITLE
Dat 4593 - global navigation layout tests

### PR DESCRIPTION
**Pull request by nikodamn**
- Add scrollbar width to desired dimension. Window is resized to that dimension although test is based on viewport size. That's why I decided to add scrollbar width there and put comment.
- Simplify method for opening dropdown component. It caused to wait everytime for 20 s.

Reviewers:
@Wikia/west-wing

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/dropdowncomponentobject/DropDownComponentObject.java [hunk 1]
```diff
@@ -54,25 +55,14 @@ public Boolean apply(WebDriver webDriver) {
...
-          }
-          return true;
-        }
-      });
-    } finally {
-      restoreDeaultImplicitWait();
+      wait.forElementClickable(entryPoint);
+      PageObjectLogging.log("DropdownClickable", "Dropdown is clickable", true, driver);
+      entryPoint.click();
+    } catch (NoSuchElementException e) {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/dropdowncomponentobject/DropDownComponentObject.java]: MAJOR Either log or rethrow this exception. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Comment by rybmat** [comment]
LGTM

**Pull request merged**